pkg/bpf/map_linux.go

```diff
@@ -214,6 +225,10 @@ func (m *Map) commonName() string {
 	return m.cachedCommonName
 }
 
+func (m *Map) nonPrefixedName() string {
+	return m.name[7:]
```

Comment:
and why `7`?
```
if len(m.name) >= 7 {
return m.name[7:]
}
return m.name
```

Reply:
Removing `cilium_` prefix but unlike it is done with the common name we want to keep things like the endpoint id in the name.

Reply:
I would suggest using something like `strings.TrimPrefix(...)` to prevent unintended behaviour here.